docs: Document how limits work on /trackedEntities [DHIS2-19050] (#1519)
dhis2/dhis2-docs: src/user/configure-programs-in-the-maintenance-app.md

| **Completed events expiry days** |  Defines the number of days for which you can edit a completed event. This means that when an event is completed and the specified number of expiry days has passed, the event is locked. <br> <br> If you set "Completed events expiry days" to 10", an event is locked ten days after the completion date. After this date you can no longer edit the event.  |
| **Expiry period type**<br> <br>**Expiry days** |  The expiry days defines for how many days after the end of the previous period, an event can be edited. The period type is defined by the expiry period type. This means that when the specified number of expiry days has passed since the end date of the previous period, the events from that period are locked. <br> <br> If you set the expiry type to "Monthly" and the expiry days to "10" and the month is October, then you can't add or edit an event to October after the 10th of November.  |
| **Minimum number of attributes required to search** |  Specify the number of tracked entity attributes that needs to be filled in to search for Tracked Entities in the Program.  |

> | **Maximum number of tracked entities to return in search** |  Specify the maximum number of tracked entities that should be returned in a search. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|

Reviewer: ```suggestion
| **Maximum number of tracked entities to return in search** |  Specify the maximum number of tracked entities that is allowed to match a search outside capture scope. This setting can be used to force the user to search in a more targeted manner and narrow down the search results, rather than allowing a user to search and see a large number of results outside their capture scope. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|
```

> | **Maximum number of tracked entity instances to return in search** |  Specify the maximum number of tracked entity instances that should be returned in a search. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|

Reviewer 2: ```suggestion
| **Maximum number of tracked entity instances to return in search** |  Specify the maximum number of tracked entities that should be returned in a search. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|
```

    type** in a *global search*. See [Configure
    Search](#configure_search) for more information.

7.  (Optional) Enter a **Maximum number of tracked entities to

>     return in search**. This specifies the amount of tracked entity

Reviewer: ```suggestion
    return in search**. This specifies the number of tracked entities
```

>     instances that will be returned in a *global search*. See [Configure

Reviewer: ```suggestion
    that will be returned in when searching outside capture scope. See [Configure
```

<!-- end list -->

  - Maximum number of tracked entities to return: This property defines how specific a 
  search must be, by limiting the number of matching tracked entities a user is allowed to get for 

>   her search criteria. If the number of matching records is larger than this maximum, they will not 

Reviewer: ```suggestion
  their search criteria. If the number of matching records is larger than this setting, they will not 
```

>   be returned. The user must provide more specific search criteria, in order to reduce the number 

Reviewer: ```suggestion
  be returned. The user must then provide more specific search criteria, in order to reduce the number 
```

  of matching records, before they are returned. More on limits 
  [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).

    > **NOTE**
    >

>     > This maximum is only applied to search results outside the users capture org unit. Within the 

Reviewer: ```suggestion
    > This maximum limit setting is only applied to search results when searching outside the users capture org unit. Within the 
```
